src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/ReleaseService.cs

```diff
@@ -123,8 +123,21 @@ public async Task SetPublishedDateAsync(Guid id)
 
         private DateTime GetNextScheduledPublishingTime()
         {
-            var publishReleasesCronSchedule = _configuration.GetSection("Values")
-                ?.GetValue<string>("PublishReleaseContentCronSchedule");
+            var publishReleasesCronSchedule = (String)null;
+            if(_configuration.GetSection("Values")?.GetValue<string>("PublishReleaseContentCronSchedule") != null)
+            {
+               publishReleasesCronSchedule = _configuration.GetSection("Values")?.GetValue<string>("PublishReleaseContentCronSchedule");
+            }
+            else
+            {
+               publishReleasesCronSchedule = _configuration.GetValue<string>("PublishReleaseContentCronSchedule");
+            }
+
+            if(publishReleasesCronSchedule == null)
+            {
+               throw new ArgumentException("PublishReleaseContentCronSchedule variable should not be null");
+            }
```

Comment:
This bit was necessary because locally, I could only get the cron string variable using .GetValue, but the tests required me to get the cron string variable using .GetSection()?.GetValue

Pretty sure this isn't The Way, but it does work...

Reply:
It looks like when building the configuration root from the local settings json file that it's not merging the Values section, so we have to GetSection("Values") instead which is nasty.

See PR https://github.com/dfe-analytical-services/explore-education-statistics/pull/1456 where I have altered the test to not build the configuration, but instead just set the environment variable for the cron string that the service uses. This also lets us do a nice test around the time 09:30.